pkg/storage/tpr/storage_interface.go

```diff
@@ -106,8 +106,18 @@ func (t *store) Create(
 		t.singularKind.URLName(),
 	).Body(data)
 
+	res := req.Do()
+	if res.Error() != nil {
+		glog.Errorf("executing DELETE for %s/%s (%s)", ns, name, res.Error())
+	}
+	var statusCode int
+	res.StatusCode(&statusCode)
+	if statusCode == http.StatusConflict {
+		return storage.NewKeyExistsError(key, 0)
+	}
```

Comment:
We already do bail on `res.Error() != nil`. I guess we could _also_ bail on unexpected status codes with a generic error, but we will need to continue to respond specifically to 404 on delete and 409 on create. Still want to handle this as a separate issue. Will create it now.